lib/__tests__/standalone-syntax.test.js

```diff
@@ -120,8 +125,12 @@ it("standalone with postcss-html syntax", () => {
     const ruleEmptyLineBeforeResult = results.find(r =>
       /[/\\]rule-empty-line-before\.html$/.test(r.source)
     );
-    expect(ruleEmptyLineBeforeResult.errored).toBeFalsy();
-    expect(ruleEmptyLineBeforeResult.warnings.length).toBe(0);
+    expect(ruleEmptyLineBeforeResult.errored).toBe(true);
+    expect(ruleEmptyLineBeforeResult.warnings.length).toBe(1);
+    expect(ruleEmptyLineBeforeResult.warnings[0].line).toBe(8);
```

Comment:
What are the 8 warnings added in this change?

Reply:
not 8 warnings, [just 1](https://github.com/stylelint/stylelint/pull/2894/files/51e7a7f7b8dfa93e0bc1dc4da241768a06eaef4f#diff-7d31f5d52b54f165816fa0ff4e0de2dfR132).